docs/config/aws_template.md

```diff
@@ -0,0 +1,440 @@
+
+2. Update the `aws-template/pipelines_config.yml` file for the new pipeline.
+
+    The second half of the `aws-template/pipelines_config.yml` file contains configurations for each deployed pipeline,
+    including the type of pipeline (i.e., `Ingest` or `VAP`), the trigger (i.e., `S3` or `Cron`), and a collection of
+    configuration files for different sites that the pipeline is deployed at (`configs` section).
+
+    ```yaml title="aws-template/pipelines_config.yml"
+    pipelines:
+    - name: lidar  # (1)!
+      type: Ingest  # (2)!
+      trigger: S3  # (3)!
+      configs:
+        humboldt:
+          input_bucket_path: lidar/humboldt/  # (4)!
+          config_file_path: pipelines/lidar/config/pipeline_humboldt.yaml # (5)!
+        morro: # (6)!
+          input_bucket_path: lidar/morro/
+          config_file_path: pipelines/lidar/config/pipeline_morro.yaml
+
+    - name: lidar_vap
+      type: VAP
+      trigger: Cron
+      schedule: Hourly  # (7)!
+      configs:
+        humboldt:
+          config_file_path: pipelines/lidar_vap/config/pipeline.yaml
+    ```
+
+    1. A useful name to give the pipeline in AWS. We recommend naming this like the folder names underneath the
+        `pipelines/` folder in the `pipeline-template` repo. E.g., if your config file is
+        `pipelines/imu/config/pipeline_humboldt.yaml`, then `imu` would be the recommended name for it.
+
+    2. The type of pipeline, either **`Ingest`** or **`VAP`**.
+
+    3. The type of trigger, either **`S3`** to trigger when a file enters the input bucket path, or **`Cron`** to run on
+        a regular schedule.
+
+    4. The subpath within the input bucket that should be watched. When new files enter this bucket, the pipeline will
+        run with those files as input.
+
+    5. The path to the pipeline configuration file in the `pipeline-template` repo.
+
+    6. Each `pipeline.yaml` config file needs to be registered so it can be deployed.
+
+        Here we define one for Morro Bay, CA in addition to the ingest for the Humboldt, CA site.
+
+        !!! note
+
+            You can keep adding new sites, or versions of this pipeline to the **`configs`** section. Just make sure
+            that the key (e.g., "morro", "humboldt") is unique for each pipeline config you add.
+
+    7. If the **`Cron`** trigger is selected, then you must also specify the schedule. The schedule should be one of the
+        following values:
+
+        * **Hourly**
+        * **Daily**
+        * **Weekly**
+        * **Monthly**
+
+3. Go to the CodePipeline UI in AWS and find the CodePipeline for this project, then click 'release change'.
+
+</div>
+
+### Updating an existing pipeline
+
+Changes to the deployed branch(es) in the `pipeline-template` repo will be released automatically via the CodePipeline
+build process in AWS, which was set up to watch for branch changes during the `./deploy_stack.sh main` step.
```

Comment:
Suggested change:

### Updating an existing Tsdat pipeline

The AWS CodePipeline build, which was created during the `./deploy_stack.sh [BRANCH_NAME]` step, will automatically watch for changes to your `pipeline-template` code in the BRANCH_NAME specified.  This means that any time you push changes to that branch, CodePipeline will automatically update and re-deploy any modified Tsdat pipelines.

Reply:
Changed this